extensions/amp-subscriptions-google/0.1/test/test-amp-subscriptions-google.js

```diff
@@ -39,6 +42,71 @@ import {toggleExperiment} from '../../../../src/experiments';
 const PLATFORM_ID = 'subscribe.google.com';
 const AMP_URL = 'myAMPurl.amp';
 
+describes.realWin('AmpFetcher', {amp: true}, env => {
+  // Please note that AmpFetcher is only called by swg-js.
+  // Also note that sendBeach accepts a message object which is private to
```

Comment:
sendBeach?

Reply:
I apparently didn't put the Message type on the function signature so this comment isn't required.